benchmark/google/view_transpose_impl.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2020 Samuel Debionne, ESRF.
+
+// Use, modification and distribution is subject to the Boost Software
+// License, Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#include <benchmark/benchmark.h>
+
+#include <boost/gil.hpp>
+
+#define GIL_ENABLE_UNROLLED 1
+
+namespace boost {
+    namespace gil {
+
+        template <typename SrcView, typename DstView>
```

Comment:
Nitpick: we don't indent content of the namespace

However, hand-rolled formatting consistency is not that important for benchmark, so it can wait for the `.clang-format` overhaul.